curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/inherit-behaviors-from-a-supertype.english.md

```diff
@@ -16,15 +16,15 @@ Animal.prototype.eat = function() {
 };
 ```
 
-This and the next challenge will cover how to reuse <code>Animal's</code> methods inside <code>Bird</code> and <code>Dog</code> without defining them again. It uses a technique called <code>inheritance</code>.
+This and the next challenge will cover how to reuse <code>Animal's</code> methods inside <code>Bird</code> and <code>Dog</code> without defining them again. It uses a technique called <dfn>inheritance</dfn>.
```

Comment:
I think we need an actual definition for `inheritance` here to be using dfn tags here.  We mention inheritance without really explaining what it means.